app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/room/RoomCategorizedPicture.kt

```diff
@@ -0,0 +1,12 @@
+package com.github.HumanLearning2021.HumanLearningApp.room
+
+import androidx.room.Embedded
+import androidx.room.Relation
+
+data class RoomCategorizedPicture(
+    @Embedded val category: RoomCategory,
```

Comment:
we don't want to embed category do we?

Reply:
I want to embed it to make the translation RoomCategorizedPicture to (Firestore)CategorizedPicture easier

Reply:
ok that makes sense